posts/Guillaume/2017-03-30-hibernate-validator-600-alpha2-out.adoc

```diff
@@ -0,0 +1,131 @@
+= "Hibernate Validator 6.0.0.Alpha2 is out with improved Bean Validation 2.0 support"
+Guillaume Smet
+:awestruct-tags: [ "Hibernate Validator", "Releases" ]
+:awestruct-layout: blog-post
+---
+
+Hibernate Validator 6 is going to be the Reference Implementation of Bean Validation 2.0. This Alpha2 release is coordinated with the Early Draft 2 release of the Bean Validation 2.0 specification.
```

Comment:
Hmmm. I presented it as such on the website and in the spec I pushed to the website.